src/router.ts

```diff
@@ -839,10 +853,15 @@ export async function determineVoterState(
   logger.debug(
     `ROUTER.determineVoterState: Passing voter message to Slack, slackChannelName: ${lobbyChannelId}, parentMessageTs: ${lobbyParentMessageTs}.`
   );
+  const blocks = SlackBlockUtil.formatMessageWithAttachmentLinks(
```

Comment:
is there any way to have this be part of SlackApiUtil.sendMessage, so that it's only called once instead of needing it in every router function?

Reply:
We could, although I think it's mixing concerns a bit. For reference, we call `SlackBlockUtil.formatMessageWithAttachmentLinks` 4 times in this file and `SlackApiUtil.sendMessage` 23 times. The other thing to note is that we're not just adding some attachments in `formatMessageWithAttachmentLinks`. It treats the underlying text format differently. In `sendMessage`, we assume the `message` is Mrkdwn and format accordingly. But `formatMessageWithAttachmentLinks` escapes all of that because it knows the text is coming from random people over Twilio. So we could move that logic into `sendMessage`, but it's a bit more than just adding an `attachments` param.

Reply:
That absolutely makes sense. Thanks for giving this a lot of thought.